stable/jenkins/README.md

```diff
@@ -68,7 +68,10 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `master.tag`                      | Master image tag                     | `lts`                                     |
 | `master.imagePullPolicy`          | Master image pull policy             | `Always`                                  |
 | `master.imagePullSecretName`      | Master image pull secret             | Not set                                   |
+| `master.disableRememberMe`        | Diable use of remember me            | `false`                                   |
+| `master.mode`                     | Jenkins mode (NORMAL or EXCLUSIVE)   | `NORMAL`                                  |
```

Comment:
this is being added as `master.executorMode` in #22283

Reply:
Removed